src/components/ButtonPanel.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => {
+  const group = [['AC', '+/-', '%', '÷'], [7, 8, 9, 'X'], [4, 5, 6, '-'], [1, 2, 3, '+'], [0, '.', '=']];
+
+  return (
+    <>
+      {group.map(elem => (
+        <div key={elem.id}>
+          {elem.map(btn => (
+            <Button key={btn.id} name={btn} />
+          ))}
+        </div>
+      ))}
+    </>
```

Comment:
I like your implementation of the ButtonPanel component! :+1: It looks short and easier to maintain! :wink: